README.md

```diff
@@ -54,6 +55,12 @@ Xamarin.Android provides open-source bindings of the Android SDK for use with
 [commercial-d15-9-macOS-x86_64-status]:   https://jenkins.mono-project.com/view/Xamarin.Android/job/xamarin-android-builds-d15-9/
 [commercial-d16-0-Windows-x86_64]:        https://download.visualstudio.microsoft.com/download/pr/b153d2f1-f6ce-4fdf-b38d-5e228ae08bd6/d308fbee786cfb26a2fd783f183f4e7e/xamarin.android.sdk.9.2.3.0.vsix
 [commercial-d16-0-macOS-x86_64]:          https://download.visualstudio.microsoft.com/download/pr/70ad99db-5313-4c32-9b0e-9c5b37a995c9/e505e4500a1a89d947906fecad034031/xamarin.android-9.2.3-0.pkg
+[commercial-d16-1-Windows-x86_64]:        https://aka.ms/xamarin-android-commercial-release-windows
```

Comment:
Shouldn't these `aka.ms` links have version numbers in them?  `xamarin-android-commercial-release-macos` sounds like it should always be the *latest* version, meaning when d16-2 becomes stable, we would "lose" the d16-1 location.  (The link for the d16-1 installer gives you the d16-2 installer.)

Reply:
The way I was picturing it is that once `d16-2` moves from Preview to Release, it will take over the `xamarin-android-commercial-release-macos` link, and `d16-1` will become a direct `https://download.visualstudio.microsoft.com` link because the `d16-1` release series will then be concluded, and the link to the latest published `d16-1` version won't be changing any more, so it won't need the extra flexibility of an intermediate redirect link.

Reply:
Though having written that down, now I wonder if users would bookmark the link from the page and try to keep using it to download `d16-1` after `d16-2` takes over the link? So maybe it would leave the least room for confusion to set up separate redirect links for each release series instead of having redirect links just for "current release" and "current preview."